.github/workflows/action-fmt.yml

```diff
@@ -0,0 +1,45 @@
+name: Run lint
+
+on:
+  workflow_call:
+    inputs:
+      os:
+        required: true
+        type: string
+
+jobs:
+  fmt:
+    runs-on: ${{ inputs.os }}
+    steps:
+      - name: Free Disk Space (Ubuntu)
```

Comment:
do the linting and building jobs need the clean up?  might make them faster without?

Reply:
I believe that by sharing artifacts we might not need the cleanup anywhere

Reply:
By the end, the job with the highest disk usage still had 13Gb free (started with 19 Gb free), so I think it's ok to skip the space reclaiming step.